property.dd

```diff
@@ -247,6 +247,25 @@ $(SECTION3 $(LNAME2 classproperties, User Defined Properties),
 	being the value it is set to.
 	)
 
+	$(P Simple getter and setter properties can be written using $(LINK2 function.html#pseudo-member, UFCS).
+	These can be enhanced with the additon of the $(CODE @property) attribute to the function, which
+	adds the following behaviors:
+	)
+
+	$(UL
+	$(LI $(CODE @property) functions cannot be overloaded with non $(CODE @property) functions with the same name.)
+	$(LI $(CODE @property) functions can only have zero, one or two parameters.)
+	$(LI $(CODE @property) functions cannot have variadic parameters.)
```

Comment:
The three points above are also mentioned later on, between the code samples. And it says "Properties may only have zero or one parameter" (not two) there.